src/monodroid/jni/embedded-assemblies-zip.cc

```diff
@@ -181,7 +182,7 @@ EmbeddedAssemblies::zip_read_cd_info (int fd, uint32_t& cd_offset, uint32_t& cd_
 		return false;
 	}
 
-	auto buf = new uint8_t[alloc_size];
+	simple_pointer_guard<uint8_t[]> buf (new uint8_t[alloc_size]);
```

Comment:
a little late, but should `alloc_size` -- declared line 178 -- be a `const size_t` or `constexpr`?  It doesn't look like it's ever changed after declaration…